website/docs/Ganglion/08-Ganglion_Data_Format.md

```diff
@@ -29,120 +29,31 @@ Once the Ganglion has initialized itself with a BLE connection it waits for comm
+The cost of this transmission strategy is a reduction in sample resolution, but this reduction is small. In the worst case scenario, only 18 bits of each 24-bit sample can be sent. This means that 6 bits of the sample are lost. This may seem problematic, but it's important to consider how the raw integer values sampled by the device convert into voltages. Each integer value corresponds to `0.001869917138805 uV` (see the [Interpreting the ExG Data](#interpreting-the-eeg-data) section for more details). The lost 6 bits of information hold the least significant, or smallest, portion of the sample. In the worst case, the sample will be off by 2^6 = 64. Converting this to volts shows that, at worst, we lose `64 * 0.001869917138805 uV = 0.11967469688352 uV` of resolution. In most applications, variations below 0.2 uV will be the result of environmental noise and no meaningful data will be lost.
 
-### 19bit Compression
+### Sample Transmission
 
-![19bit Packet Map](../assets/GanglionImages/ganglion_19bit-packet-map.png)
-
-By default we use a 19-bit delta compression. See below for the story about 18bit delta compression description. It's easiest to understand this compression scheme if we step through the process. Assume that the Ganglion is connected to a computer, and the user wants to start streaming data. The user (or controlling software) sends a `b`, and when the Ganglion receives the `b`, it turns on the data acquisition hardware, starts taking samples, and sending data in 20 byte packets. The very first byte of each sample is the `Packet ID`, and the very first packet sent starts with an ID of `0x00`. The value `0x00` indicates to the controlling software that the packet contains **Uncompressed Data**, that is, there are 4 24bit values taking up the 12 bytes following the packet ID associated with channel 1-4 respectively. These values are legit Ganglion data, and are used to seed the decompression algorithm on the computer. The Ganglion stores the values in an array called `lastChannelData[]`.  
-**NOTE:** You can find all of this code in our [Ganglion Library](https://github.com/OpenBCI/OpenBCI_Ganglion_Library) repository.
+We send ExG samples using either 19 bits or 18 bits. By default, 19-bit samples are sent. When the accelerometer or auxiliary features of the board are used, 18-bit samples are sent to provide room for the accelerometer or auxiliary data in the packet. Different packet ID ranges are used for 19-bit samples and 18-bit samples. This way, the controlling software that is running the decompression algorithm can decompress each packet without having to 'know' what state the system is in.
 
-```
+**IMPORTANT!** Both the 18-bit and 19-bit transmission strategies store the signed bit in the `bit 0` or LSB position!
 
-void OpenBCI_Ganglion::compressData19() {
```

Comment:
Why were these large code blocks removed?

Reply:
Because they're no longer valid. We don't compress data that way anymore.